doc/source/management/data_at_rest_encryption.rst

```diff
@@ -363,33 +683,73 @@ System Variables
 
 .. variable:: keyring_vault_config
 
-  :version 5.7.20-18: Implemented
-  :cli: ``--keyring-vault-config``
-  :dyn: Yes
-  :scope: Global
-  :vartype: Text
-  :default:
+   :version 5.7.20-18: Implemented
+   :cli: ``--keyring-vault-config``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Text
+   :default:
 
 This variable is used to define the location of the
 :ref:`keyring_vault_plugin` configuration file.
 
 .. variable:: keyring_vault_timeout
 
-  :version 5.7.21-20: Implemented
-  :cli: ``--keyring-vault-timeout``
-  :dyn: Yes
-  :scope: Global
-  :vartype: Numeric
-  :default: ``15``
+   :version 5.7.21-20: Implemented
+   :cli: ``--keyring-vault-timeout``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Numeric
+   :default: ``15``
 
 This variable allows to set the duration in seconds for the Vault server
 connection timeout. Default value is ``15``. Allowed range is from ``1``
 second to ``86400`` seconds (24 hours). The timeout can be also completely
 disabled to wait infinite amount of time by setting this variable to ``0``.
 
-Other reading
--------------
+Data Scrubbing
+================================================================================
+
+While data encryption ensures that the existing data are not reachable in
+plain form, the data scrubbing removes the data once the user decides they
+should be deleted. Also, data scrubbing removes copies of data from logs.
+
+Once enabled, data scrubbing works automatically on each tablespace
+separately. To enable data scrubbing, you need to set the following variables:
+
+- :variable:`innodb-background-scrub-data-uncompressed`
+- :variable:`innodb-background-scrub-data-compressed`
```

Comment:
These variables only control tablespace scrubbing, while the paragraph above mentions logs too. Log scrubbing is controlled by a different variable, mentioned earlier. (and similarly to the immediate data scrubbing, it isn't related to the encrpytion threads).

Backround scrubbing also won't do anything when there aren't any encryption threads, as it is executed by those thereads - so it needs that variable >0.

Reply:
Text updated